### PR TITLE
And then there were tests

### PR DESCRIPTION
**Pull request by bfpepper**
Adds test for about page.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -14,30 +14,23 @@ def create_timezones
...
-        slack_username: "@mentor#{n}",
-        location: Faker::Address.city,
-        mentor_timezone_id: MentorTimezone.find(Random.new.rand(1..4)).id,
-        bio: 'This is my mentor bio.',
-        expertise: Faker::Lorem.words(6).join(", "),
+      puts "Crated user: #{n}!"
+      user.create_mentor!(
+        timezone_id: timezone.id,
+        expertise: "Rails",
+        location: Faker::LordOfTheRings.location,
```
Comment (meganft) [inline db/seeds.rb]: 👍

File: spec/features/mentor_show_page_spec.rb [hunk 2]
```diff
@@ -3,12 +3,13 @@
...
-      Mentor.create(avatar: "string", name: "Frank", email: "This@gmail.com", phone_number: "222222222", slack_username: "slackguy82", location: "Denver", bio: "Soft ware dev with now thinz bouts codessss", expertise: "all", company: "fan", position: "dev pro", mentor_timezone_id: MentorTimezone.first.id, last_active:DateTime.now , profile_completed: true)
+      timezone = create(:timezone)
+      user_1, user_2 = create_list(:user, 2)
+      mentor_1, = create(:mentor, user: user_1, timezone: timezone)
+      mentor_2, = create(:mentor, location: "Denver", user: user_2, timezone: timezone)
       visit '/mentors/1'
-      expect(page).to have_content("Bob")
-      expect(page).to_not have_content("Frank")
+      expect(page).to have_content("#{mentor_1.location}")
+      expect(page).to_not have_content("#{mentor_2.location}")
```
Comment (meganft) [inline spec/features/mentor_show_page_spec.rb]: Looks good @bfpepper

**Comment by jbkimble** [comment]
Updated tests look good!  There seems to be some non-test work done that might belong on a different branch in the future. 👍

**Comment by vidoseaver** [comment]
dated. removed.

**Comment by vidoseaver** [comment]
no

**Pull request closed without merging**